docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 1
+---
+
+# Postgres Destination
+
+In order to create an end-to-end test of the ELT pipeline apart from the Source which is the KYVE network, you should
+configure a destination where the `records` will be stored.
```

Comment:
same comment here as in #49 -- I think the sentence is a bit confusing and could maybe be dropped, since the next sentence makes clear what the page is about?

Reply:
That makes sense, I'll remove it.